content/ytt/docs/latest/_index.md

```diff
@@ -1,53 +1,85 @@
 ---
 title: "About ytt"
-toc: "false"
 cascade:
   version: latest
   toc: "true"
   type: docs
   layout: docs
 ---
 
-`ytt` is a templating tool that understands YAML structure. It helps to easily configure complex software via reusable templates and user provided values.
+## Overview
 
-A comparison to similar tools can be found in the [ytt vs X doc](ytt-vs-x.md).
+`ytt` is a command-line tool used to template and patch YAML files. It also provides the means to collect fragments and piles of YAML into modular chunks for easy re-use.
+
+In practice, these YAML files are [Kubernetes configuration](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/), [Concourse Pipeline](https://concourse-ci.org/pipelines.html#schema.pipeline), [Docker Compose](https://github.com/compose-spec/compose-spec/blob/master/spec.md#compose-file), [GitHub Action workflow](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions) files..., really anything that is in YAML format.
+
+`ytt` is most useful when manually maintaining these files has or will become too much work.
```

Comment:
imho this sentence is confusing. i know what you are saying but i dont think it reads that way.

Reply:
Thank you. 🙏🏻 

Any suggestions? Actionable observations?